### ISSUE TITLE
Add option to send keystroke to STV when screen saver is deactivated.

### ISSUE THREAD

**Issue by tmiranda1962** [google/sagetv#29]
When the user presses a key and deactivates the screen saver, the keystroke is never passed to the STV.  I'd like to see that keystroke passed on to the STV.  It's probably better to make this an option to ensure backwards compatibility.

**Comment by CraziFuzzy** [comment]
I think maybe SOME keys should be passed through, but I'm not sure about all.  Some things are not to be done blind.  I can see maybe just Play passing though (thinking the screen saver has kicked in while a show is paused, so coming back and hitting play will exit the screen saver and continue on - same with maybe HOME.  But things like number keys, Favorite/Don't Lie/Delete, etc should not be passed on.

**Comment by tmiranda1962** [comment]
That will work. My main reason for wanting this is so I complete my 'tag
photos' plugin. That plugin allows the user to 'tag' photos while they are
being shown on the screen saver.

**Comment by CraziFuzzy** [comment]
Sounds like the best thing would be to have your plugin modify the screen saver to grab it directly, since what you are really doing is altering the screen saver's behavior.  Not sure this needs to be a core change in the STV.

**Comment by tmiranda1962** [comment]
The core "swallows" the keystroke and never passes it on to the STV.  I've
spoke to Jeff about this in the past and he said it would require a core
change but didn't want to do it until Sage was open sourced.

**Comment by Narflex** [comment]
Yeah, I'm cool with a change like this...but of course as you mention, by default it should not get passed to the STV for compatibility. Since this is an STV specific behavior, I'd encourage you to use a global attribute in the STV for controlling this rather than a property.  Search the code for "SingularMouseTransparency" to see an example of this. (and feel free to modify the updateXBMCCompatible function to have a more generic name while you're in there)
